<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QPushButton" name="loadDataButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="labelDataRange_pushButton">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>470</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Label Data Range</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>541</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="PlotWidget" name="whole_plot" native="true"/>
         </item>
         <item>
          <widget class="PlotWidget" name="crop_plot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBoxTime_label">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>540</y>
          <width>301</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="frameSize_label">
           <property name="text">
            <string>3 sec</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Playback Speed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="playbackSpeedSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="playbackspeed_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Frame Size</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="frameSizeSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="DataLabels_comboBox">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>470</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>581</width>
          <height>101</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="stepBackButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt; step back</string>
         </property>
        </widget>
        <widget class="QPushButton" name="playPauseButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Play / Pause</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stepFwdButton">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>step fwd &gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="restartPlayButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
        <widget class="QLabel" name="frameNumber_label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QSlider" name="playbackPositionSlider">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>521</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="filename_label">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>120</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
       <widget class="QTreeView" name="treeView">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>10</y>
          <width>256</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="plotButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>120</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>200</y>
          <width>341</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>350</y>
          <width>151</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="saveTimeStamsOnly_radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Save timestamps only</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="saveCroppedData_radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Save cropped  data</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>350</y>
          <width>191</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop Time</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="startTime_lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="stopTime_lineEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Start Time</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>440</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>624</x>
          <y>510</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt; Previous Selection</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>510</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next Selection &gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
